tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientProvider.java

```diff
@@ -14,6 +15,17 @@
     private ManagementClientProvider() {
     }
 
+    /**
+     * Creates {@link OnlineManagementClient} for <b>standalone</b> mode, based on {@link ArquillianContainerProperties}
```

Comment:
Maybe here you could mention that the descriptor is obtained by static `ArquillianDescriptorWrapper` reference. WDYT?

Reply:
Right, fixed and repushed.